nengo_gui/components/pointer.py

```diff
@@ -16,17 +15,35 @@ def __init__(self, obj, **kwargs):
         # the semantic pointer value as set by the user in the GUI
         # a value of 'None' means do not override
         self.override_target = None
-        self.vocab_in = obj.inputs[self.target][1]
+        loop_in_whitelist = [spa.Buffer, spa.Memory, spa.State]
```

Comment:
There should be a comment indicating what this whitelist is, and what conditions there are if someone were to add something new onto the whitelist (and also why we have this whitelist in the first place)